src/Symfony/Component/Validator/Context/ExecutionContextInterface.php

```diff
@@ -66,10 +66,11 @@ interface ExecutionContextInterface
      *
      * @param string|\Stringable $message The error message as a string or a stringable object
      * @param array              $params  The parameters substituted in the error message
+     * @param string|false|null  $translationDomain The translation domain of the error message
```

Comment:
see fabbot for CS :)

Reply:
Can you help me on this @nicolas-grekas, 
You asked for `@param` annotation here: https://github.com/symfony/symfony/pull/48852#discussion_r1164231899
but the fabbot is asking me to remove them. So what should I do ?

Reply:
you should align the `The`

Reply:
See https://github.com/symfony/symfony/pull/48852/commits/fcc4024b8e29c08ea85e77d16c384b555fa010e5, it's aligned

The fabbot is asking for 
<img width="898" alt="image" src="https://user-images.githubusercontent.com/9052536/231702067-05c95da3-e966-4a92-9456-1c355bb04bd3.png">

So I feel like I have to remove the phpdoc

Reply:
Nobody has to obey to bots, we still are the ones with a brain :)
This is resolved then.